libraries/standard/mqtt/cbmc/include/mqtt_cbmc_state.h

```diff
@@ -18,11 +18,92 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
+#ifndef MQTT_CBMC_STATE_H_
+#define MQTT_CBMC_STATE_H_
+
+#include <stdbool.h>
+
+#include "mqtt.h"
```

Comment:
Not sure you want the project header file pulled into a proof header file, but probably okay

Reply:
This is for the MQTT struct types